### DISCUSSION TITLE
Snap to Grid

### DISCUSSION THREAD

**Discussion by michelmar** [stevenpetryk/mafs#110] [Help] [answered]
Is it possible to snap moveable points only to the grid?

**Comment by stevenpetryk** [comment, marked as answer]
Yep, check out the _Constraints_ section of the [Movable Point docs](https://mafs.dev/guides/interaction/movable-points). There's an [example](https://stackblitz.com/edit/react-mafwhz?file=src%2Findex.tsx) there that snaps to the grid.
